### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by ozasadnyy**
## Mobile

<br/>

### CODE REVIEW

File: scripts/schedule-webworker.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+function getEndTime(date, startTime, endTime, totalNumber, number) {
+  const timezone = new Date().toString().match(/([A-Z]+[\+-][0-9]+.*)/)[1];
+  const timeStart = new Date(date + ' ' + startTime + ' ' + timezone).getTime();
```
Comment (sophieH29) [inline scripts/schedule-webworker.js]: How about
`${date} ${startTime } ${timezone}`

File: scripts/schedule-webworker.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  const timeEnd = new Date(date + ' ' + endTime + ' ' + timezone).getTime();
+  const difference = Math.floor((timeEnd - timeStart) / totalNumber);
+  const result = new Date(timeStart + difference * number);
+  return result.getHours() + ':' + result.getMinutes();
+}
+
+function getDuration(date, startTime, endTime) {
+  const timezone = new Date().toString().match(/([A-Z]+[\+-][0-9]+.*)/)[1];
+  const timeStart = new Date(date + ' ' + startTime + ' ' + timezone).getTime();
+  const timeEnd = new Date(date + ' ' + endTime + ' ' + timezone).getTime();
```
Comment (sophieH29) [inline scripts/schedule-webworker.js]: timezone, timeStart and timeEnd are also calculated infunction above. Better to move this into common function

File: src/pages/schedule-page.html [hunk 3]
```diff
@@ -1,24 +1,94 @@
...
+      data="{{routeData}}"
+      query-params="{{queryParams}}"
+    ></app-route>
+
+    <div class="container">
+      <iron-pages selected="[[subRoute]]" attr-for-selected="name">
+        <template is="dom-repeat" items="[[transformToArray(schedule.days, 'date')]]" as="day">
+          <schedule-day
+            name$="[[day.date]]"
+            day="[[day]]"
```
Comment (sophieH29) [inline src/pages/schedule-page.html]: don't you need day$=

File: scripts/schedule-webworker.js (hunk 1 above)
Comment (ozasadnyy) [inline scripts/schedule-webworker.js]: True, copy-paste from previous app :)

File: src/pages/schedule-page.html (hunk 3 above)
Comment (ozasadnyy) [inline src/pages/schedule-page.html]: no, we don't need to have this information as an attribute, just passing the object to the children

**Pull request merged**